nulib/meadow
# Replace Sequins with local Broadway

# Summary 

Cut out SNS entirely and dispatch straight to SQS

# Specific Changes in this PR
- Remove Sequins dependency
- Add `Meadow.Pipeline.Action` (copied from `Sequins.Pipeline.Action` but without the SNS bits)
- Update actions and tests to use `Meadow.Pipeline.Action`
- Change `sequins` to `pipeline` in configs, mix tasks, and terraform files

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

Create Works and FileSets via both the UI and ingest sheets and make sure they process as expected.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes
- [x] Pipeline configuration changes
  - All handled by Terraform; no `meadow.pipeline.setup` needed


# Tested/Verified
